skema/skema-rs/mathml/src/ast/operator.rs

```diff
@@ -6,28 +6,36 @@ use std::fmt;
 use utoipa::ToSchema;
 use schemars::JsonSchema;
 
-/// Total Derivative operator, in line with Spivak notation: http://ceres-solver.org/spivak_notation.html
+
+/// Total Derivative operator, e.g. dS/dt . in line with Spivak notation: http://ceres-solver.org/spivak_notation.html
 #[derive(Debug, Ord, PartialOrd, PartialEq, Eq, Clone, Hash, new, Deserialize, Serialize, JsonSchema)]
 pub struct Derivative {
     pub order: u8,
     pub var_index: u8,
     pub bound_var: Ci,
 }
 
-/// Partial derivative operator
+/// D Derivative operator, e.g. DS/Dt . in line with Spivak notation: http://ceres-solver.org/spivak_notation.html
+#[derive(Debug, Ord, PartialOrd, PartialEq, Eq, Clone, Hash, new, Deserialize, Serialize)]
+pub struct DDerivative {
```

Comment:
The `DDerivative` operator should be combined with the `Derivative` operator --- unless I am mistaken, their semantics are the same.